cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -76,7 +77,6 @@ parameter_types! {
 
 parameter_types! {
 	pub const CreateAssetCall: [u8;2] = [53, 0];
-	pub const CreateAssetDeposit: u128 = (UNITS / 10) + EXISTENTIAL_DEPOSIT;
```

Comment:
I removed the `EXISTENTIAL_DEPOSIT` from the CreateAssetDeposit since the Snowbridge sovereign is pre-funded. I don't have strong opinions about this though, I am happy to put it back. The Polkadot runtime config does not add the existential deposit though.